sspl_test/alerts/os/test_systemd_service_actuator.py

```diff
@@ -0,0 +1,143 @@
+from sspl_test.rabbitmq.rabbitmq_ingress_processor_tests import RabbitMQingressProcessorTests
+from sspl_test.rabbitmq.rabbitmq_egress_processor import RabbitMQegressProcessor
+from sspl_test.common import check_sspl_ll_is_running
+
+RESOURCE_TYPE = "node:sw:os:service"
+
+def init(args):
+    pass
+
+def test_systemd_service_valid_request(args):
+    check_sspl_ll_is_running()
+    # TODO: Change service name, once get final 3rd party service name
+    service_actuator_request("rsyslog.service", "status")
+    service_actuator_msg = None
+    time.sleep(6)
+    ingressMsg = {}
+    while not world.sspl_modules[RabbitMQingressProcessorTests.name()]._is_my_msgQ_empty():
+        ingressMsg = world.sspl_modules[RabbitMQingressProcessorTests.name()]._read_my_msgQ()
+        time.sleep(0.1)
+        print("Received: %s " % ingressMsg)
+        try:
+            # Make sure we get back the message type that matches the request
+            msg_type = ingressMsg.get("actuator_response_type")
+            if msg_type["info"]["resource_type"] == RESOURCE_TYPE:
+                service_actuator_msg = msg_type
+                break
+        except Exception as exception:
+            time.sleep(0.1)
+            print(exception)
+
+    assert(service_actuator_msg is not None)
+    assert(service_actuator_msg.get("alert_type") is not None)
+    assert(service_actuator_msg.get("severity") is not None)
+    assert(service_actuator_msg.get("host_id") is not None)
+    assert(service_actuator_msg.get("info") is not None)
+
+    info = service_actuator_msg.get("info")
+    assert(info.get("site_id") is not None)
+    assert(info.get("node_id") is not None)
+    assert(info.get("cluster_id") is not None)
+    assert(info.get("rack_id") is not None)
+    assert(info.get("resource_type") == RESOURCE_TYPE)
+    assert(info.get("event_time") is not None)
+    assert(info.get("resource_id") is not None)
+
+    assert(service_actuator_msg.get("specific_info") is not None)
+
+def test_systemd_service_invalid_request(args):
+    check_sspl_ll_is_running()
+    service_actuator_request("temp_dummy.service", "status")
+    service_actuator_msg = None
+    time.sleep(6)
+    ingressMsg = {}
+    while not world.sspl_modules[RabbitMQingressProcessorTests.name()]._is_my_msgQ_empty():
+        ingressMsg = world.sspl_modules[RabbitMQingressProcessorTests.name()]._read_my_msgQ()
+        time.sleep(0.1)
+        print("Received: %s " % ingressMsg)
+        try:
+            # Make sure we get back the message type that matches the request
+            msg_type = ingressMsg.get("actuator_response_type")
+            if msg_type["info"]["resource_type"] == RESOURCE_TYPE:
+                service_actuator_msg = msg_type
+                break
+        except Exception as exception:
+            time.sleep(0.1)
+            print(exception)
+
+    assert(service_actuator_msg is not None)
+    assert(service_actuator_msg.get("alert_type") is not None)
```

Comment:
we can start asserting for correct values now, for all the asserts, ofcourse where ever possible, like we cant do for event_time

Reply:
Added actual value check for for alert_type, resource_id, resurce_type.